src/base/async/int.c

```diff
@@ -1674,7 +1674,9 @@ void redirect_devices(void)
  */
```

Comment:
I am puzzled with the subject of this,
what do you mean by
"Change to pass redirector not DOS version"?
Otherwise looks good.